### PR TITLE
Cache disabled by default

### PR DESCRIPTION
**Pull request by paganotoni**
Disables the cache by default to avoid the memory leak issue mentioned in #118.

### CODE REVIEW

File: plush.go [hunk 1]
```diff
@@ -32,24 +33,25 @@ func BuffaloRenderer(input string, data map[string]interface{}, helpers map[stri
...
+
 	return t.Exec(NewContextWith(data))
 }
 
 // Parse an input string and return a Template, and caches the parsed template.
 func Parse(input string) (*Template, error) {
 	moot.Lock()
 	defer moot.Unlock()
-	if t, ok := cache[input]; ok {
+	t, ok := cache[input]
```
Comment (saurori) [inline plush.go:48]: The mutex and cache checking can be avoided altogether if cache is disabled. Adds some redundant code but you could do at top of method:
```
if !cacheEnabled {
    return NewTemplate(input)
}
```
Then all the other `cacheEnabled` checks can be removed.

Comment (paganotoni) [reply plush.go:48]: Yes, you're right. Thanks for it.

Comment (paganotoni) [reply plush.go:52]: Updated.

**Review by saurori — APPROVED** [review]
LGTM. Some tests around the caching would be good.

**Comment by paganotoni** [comment]
Added some tests. @stanislas-m LMK if this make sense for you.

**Pull request merged**